src/components/Header.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import bblfshLogo from '../img/babelfish_logo.svg';
+import githubIcon from '../img/github.svg';
+
+const Container = styled.header`
+  height: 70px;
+  padding: 0 1rem;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid black;
+`
+
+const Title = styled.h1`
+  display: flex;
+  align-items: center;
+`
+
+const TitleImage = styled.img`
+  height: 40px;
+`
+
+const DashboardTitle = styled.span`
+  font-weight: bold;
+  font-size: 1.5rem;
+  margin-left: .5rem;
+`
+
+const Actions = styled.div`
+  width: 100%;
+  display: flex;
+  margin-left: 1rem;
+`
+
+const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+
+  & + & {
+    margin-left: 1rem;
+  }
+
+  & > label {
+    margin-right: .5rem;
+  }
+`
+
+const RunButton = styled.button`
+  padding: .2rem 1.5rem;
+  border-radius: 3px;
+  border: none;
+  cursor: pointer;
+`
+
+const DriverCodeBox = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: .5rem;
+`
+
+const DriverCodeText = styled.a`
+  margin-left: .5rem;
+`
+
+export function DriverCode({ languages, selectedLanguage, actualLanguage }) {
+  const driver = selectedLanguage === 'auto' 
+    ? actualLanguage
+    : selectedLanguage;
```

Comment:
Well, they should have thought about it when they made `if` a statement and not an expr.

Anyway, was that a rant about multiline ternary ops or do you want me to rewrite it into a full-fledged if?